README.md

```diff
@@ -9,6 +9,7 @@ A Modern MP3 Player
 $ git clone https://github.com/sanket143/MJam.git
 $ cd MJam
 $ npm install
+$ npm install -g electron
```

Comment:
We don't need to install electron explicitly as it is mentioned as a dependency in `package.json`.

`$ npm install` will install electron too.

Reply:
When I followed the instructions provided in the README, I got an error saying electron could not be found. I believe that either the readme needs updated, or the npm start script needs to be updated to use the version of electron in the node_modules folder rather than a global use (current usage)

Reply:
It takes time to install electron. It actually downloads some files and install them. If you interrupt the installation, those files may not have completely installed.

You need to make sure whether `npm install` executed successfully without any interruption. You could try again by removing the `node_modules` folder. Also, `npm install -g electron` will fetch the latest `electron` version and not the one mentioned in the `package.json`, it may orginate inconsistency when there are a lot of contributors.

Reply:
Thanks for the clarification. It looks like it worked perfectly on my Mac. In this case, I'd say we are good to close this PR.